openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -127,6 +128,12 @@ def test_get_account_empty_string(self):
         ),
         ("client", "user", "country", "GB", "XY", "Select a valid choice. XY is not one of the available choices."),
         ("client", "user", "year_of_birth", 2009, "not_an_int", "Enter a whole number."),
+        (
+            "client", "user", "name", "bob",
+            "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz",
+            "Ensure this value has at most 255 characters (it has 256)."
+        ),
+        ("client", "user", "name", "Tom Jones", "z   ", "The name field must contain at least one character."),
```

Comment:
Maybe add a comment in this list that email is tested below because these tests assume that the value is changed immediately. I initially thought that you'd just missed it from this list.